src/App.js

```diff
@@ -15,6 +16,7 @@ function App() {
     <div className="flex flex-col gap-12 w-full">
       <NavBar />
       <Outlet />
+      <Footer />
```

Comment:
The component is almost going out of the viewport.
Please add some padding or margin to give some space.